jsons/parameters.json

```diff
@@ -2427,6 +2427,20 @@
         "value_translations": {},
         "value_type": "int"
     },
+    {
+        "default_value": 5,
+        "description": "absolute number of non-NA data points in which ions were detected",
```

Comment:
Is this the minimum number of allowed na data points?

Reply:
in principle its just an int defining how many non-NA values you "accept". depending on the node it can be the min or max allowed NAs, so I kept it general. In my case the two nodes I added use it as a min value, hence the node param name "min_cmatrix_ions"